pod/video/templates/channel/channel.html

```diff
@@ -95,13 +95,18 @@ <h2 class="parent_title">{{title}}</h2>
         <div class="themes-videos">
             <!-- THEME SECTION -->
             <section class="theme-section">
-                <p class="theme-count">{% blocktrans count counter=count_themes %}{{ counter }} theme found{% plural %}{{ counter }} themes found{% endblocktrans %}</p>
+                <p class="theme-count">
+                    {% blocktrans count counter=count_themes %}{{ counter }} theme found{% plural %}{{ counter }} themes found for this channel{% endblocktrans %}
```

Comment:
Ici je pense qu'il faut ajouter aussi "for this channel" dans la partie au singulier.